id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java

```diff
@@ -163,6 +163,11 @@ public ResponseWrapper<CredentialIssueResponse> createCredentialIssuanceByRid(Cr
 
 		CredentialIssueResponse credentialIssueResponse = null;
 		try{
+			Optional<CredentialEntity> entity = credentialDao.findById(rid);
+			if (entity.isPresent()) {
+				LOGGER.debug(IdRepoSecurityManager.getUser(), CREDENTIAL_SERVICE, CREATE_CREDENTIAL,

```

Comment:
@Neha2365 , you will need to use + to append multiple texts. if you use , it is expected that the first text is having placeholder (%s) for the next following arguments.

Reply:
Also please add a logger for else block also.